src/ui/components/Timeline/UnloadedRegions.tsx

```diff
@@ -29,8 +31,9 @@ export const UnloadedRegions: FC = () => {
   let beginTime = begin.time;
   let endTime = end.time;
   if (focusRegion) {
-    beginTime = startTimeForFocusRegion(focusRegion);
-    endTime = endTimeForFocusRegion(focusRegion);
+    const focusedAndLoaded = overlap([rangeForFocusRegion(focusRegion)], loadedRegions.loading)[0];
+    beginTime = focusedAndLoaded.begin.time;
+    endTime = focusedAndLoaded.end.time;
```

Comment:
I think this might be a potential bug scenario. If `overlap` doesn't return at least (exactly?) one item, `focusedAndLoaded` will be undefined.

Reply:
I think [this recording](https://app.replay.io/recording/hit-counts-are-off-typeerror--8759e652-2916-4453-9510-59615c9389e1) shows that scenario, where the times passed in are 12-48s and 50-110s (which don't overlap).

Reply:
Potential fix in #7202 – though I haven't thoroughly tested it.